assessments/KRAssessment/seeds/academicSubjectDescriptors.csv

```diff
@@ -0,0 +1,6 @@
+codeValue,description,namespace,shortDescription
+SocialFoundations,Social Foundations,uri://ed.sc.gov/AcademicSubjectDescriptor,Social Foundations
```

Comment:
These will need to exist in the ODS already for these to send successfully

Reply:
also, when we let Wendy know about these new academic subject descriptors, you can include typical spacing between words. But there is going to be a length constraint of 50 chars